src/style.css

```diff
@@ -1,3 +1,102 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Arial, sans-serif;
+  background-color: grey;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
+.nav {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  line-height: 60px;
+  height: 60px;
+  margin-right: 50px;
+}
+
+@media (width: 1200px) {
+  .img-link {
+    margin-right: 419px;
+  }
+}
+
+@media (width: 1024px) {
+  .img-link {
+    margin-right: 243px;
+  }
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  text-transform: uppercase;
+  height: 60px;
+}
+
+.img {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.img-link {
+  padding-left: 50px;
```

Comment:
The `padding-left` in the `.img-link` class might prevent the content from being centered vertically. Consider adjusting this padding to ensure the content is centered as required.